components/fastly_frontend/main.tf

```diff
@@ -155,11 +169,29 @@ resource "fastly_service_v1" "frontend" {
     content = file("${path.module}/app_name.vcl")
   }
 
+  # Configure 'robots.txt' global deny for backends:
+  condition {
+    type      = "REQUEST"
+    name      = "path-robots"
+    statement = "req.url.basename == \"robots.txt\""
+  }
+
+  dynamic "response_object" {
+    # If we're not in production, add a 'robots.txt' to deny web crawlers.
+    for_each = var.environment != "production" ? [1] : []
```

Comment:
I just remembered to add this in ff77c81! 🙈 This keeps our same "deny all" `robots.txt` in development and QA to prevent those environments from showing up in Google search results. 

Terraform doesn't have a simple "conditional" syntax for blocks so, [like we do with the `count` attribute for resources](https://github.com/DoSomething/infrastructure/blob/13f1ec7367067a89746f605e75704aaa95eb397d/components/s3_bucket/main.tf#L113), we use a zero/one item "for loop" to make this conditional on `var.environment`.